src/esp32/base.rs

```diff
@@ -128,11 +128,14 @@ where
     }
 }
 
-impl<ML, MR> Stoppable for Esp32WheelBase<ML, MR>
+impl<ML, MR> Actuator for Esp32WheelBase<ML, MR>
 where
     ML: Motor,
     MR: Motor,
 {
+    fn is_moving(&mut self) -> anyhow::Result<bool> {
+        Ok(self.motor_left.is_moving()? || self.motor_right.is_moving()?)
+    }
```

Comment:
Not part of this review, but it is interesting to me that for `stop` here, if we fail to stop the `motor_left` motor, we don't even try to stop the `motor_right` one.

Reply:
yeah, this seems bad. I'll make a ticket for this